.ci/Manifest.toml

```diff
@@ -245,9 +250,9 @@ version = "1.1.0"
 
 [[RegistryTools]]
 deps = ["AutoHashEquals", "LibGit2", "Pkg", "UUIDs"]
-git-tree-sha1 = "94cf95fd962cc0594da4d676ed66b553f84ec246"
+git-tree-sha1 = "3dfa671318ac8af835cc621b08fcdcc28a4aee67"
 uuid = "d1eb7eb1-105f-429d-abf5-b0f65cb9e2c4"
-version = "1.3.4"
```

Comment:
This shouldn't be downgraded, see https://github.com/JuliaPackaging/Yggdrasil/pull/777

Reply:
Good catch; the resolver did that to me.  I guess we should force a compat entry here.

Reply:
sigh, old registries for the fail